### DISCUSSION TITLE
Unable to use GitHub Codespaces for VS

### DISCUSSION THREAD

**Discussion by sr229** [community/community#3718] [Codespaces] [answered]
For some reason trying to use GitHub Codespaces for VS doesn't work since I don't have the API scope, but I recently got cleared for access for this. I already raised a GitHub Support Ticket to help me out with this.

**Comment by 2percentsilk** [comment, marked as answer]
Hi there,

As of April 12th, connecting to GitHub Codespaces from Visual Studio 2019 will no longer be supported and this private preview has [concluded](https://docs.microsoft.com/en-us/visualstudio/ide/codespaces/codespaces-overview?view=vs-2019).

**Reply by sr229** [reply]
Thanks for answering, does that mean VS won't get a similar product in the near future?
